docs/api/orders.rst

```diff
@@ -512,5 +649,140 @@ Example Response
             }
         ],
         "currency_code":"USD",
+        "locale_code":"en_US",
         "checkout_state":"completed"
     }
+
+Ship Action
+-----------
+
+You can ship the placed order by executing the following request:
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    PUT /api/v1/orders/{orderId}/shipments/{id}/ship
+
++---------------+----------------+-----------------------------------------------+
+| Parameter     | Parameter type | Description                                   |
++===============+================+===============================================+
+| Authorization | header         | Token received during authentication          |
++---------------+----------------+-----------------------------------------------+
+| orderId       | url attribute  | Id of requested order                         |
++---------------+----------------+-----------------------------------------------+
+| id            | url attribute  | Id of shipment to ship                        |
++---------------+----------------+-----------------------------------------------+
+| tracking      | request        | *(optional)* Tracking code of shipment if any |
++---------------+----------------+-----------------------------------------------+
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/orders/21/shipments/21/ship \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json" \
+        -X PUT
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    STATUS: 204 No Content
+
+.. note::
+
+    It is important to emphasize that in this example the shipment id has the same value as for order, but it is an accident rather then rule
```

Comment:
`It is important to emphasise that in this example the shipment id is the same value as for the order, but it is a coincidence rather than a rule.`